features/support/env.rb

```diff
@@ -0,0 +1,12 @@
+require 'cucumber/rails'
+
+ActionController::Base.allow_rescue = false
+
+begin
+  DatabaseCleaner.strategy = :transaction
+rescue NameError
+  raise "You need to add database_cleaner to your Gemfile (in the :test group) if you wish to use it."
+end
+
```

Comment:
please add this line 
`World FactoryBot::Syntax::Methods
`

Reply:
added together with fixes in gemfile and env.rb